planner/core/logical_plan_builder.go

```diff
@@ -1560,6 +1560,19 @@ func (t *itemTransformer) Leave(inNode ast.Node) (ast.Node, bool) {
 	return inNode, false
 }
 
+func (b *PlanBuilder) checkOrderByItems(ctx context.Context, p LogicalPlan, byItems []*ast.ByItem, aggMapper map[*ast.AggregateFuncExpr]int, windowMapper map[*ast.WindowFuncExpr]int) error {
```

Comment:
Please add some comments to explain what this function wants to do. If we do not need `sort`, why do we need to handle the `byItems`?

Reply:
Here we have check for errors like `ErrBadField` to be compatible with MySQL.